src/components/views/rooms/MessageComposer.js

```diff
@@ -149,7 +149,10 @@ const EmojiButton = ({addEmoji}) => {
         const buttonRect = button.current.getBoundingClientRect();
         const EmojiPicker = sdk.getComponent('emojipicker.EmojiPicker');
         contextMenu = <ContextMenu {...aboveLeftOf(buttonRect)} onFinished={closeMenu} catchTab={false}>
-            <EmojiPicker onChoose={addEmoji} showQuickReactions={true} />
+            <EmojiPicker onChoose={(emoji) => {
```

Comment:
Using an inline function inside JSX like [can be bad for performance](https://reactjs.org/docs/faq-functions.html#arrow-function-in-render), as it means the prop's value change every time, potentially causing the component and its children to re-render unnecessarily.

For a class based component, the typical fix is to define the event handler as a method on the class so it doesn't change. For a hooks-based component like this one, you can use [`useCallback`](https://reactjs.org/docs/hooks-reference.html#usecallback) to memoise the handler.

Reply:
Sorry for that, I didn't realize this could have a performance effect. Should be fixed now